test/functional/apps/visualize/_tsvb_chart.js

```diff
@@ -28,28 +28,56 @@ export default function ({ getService, getPageObjects }) {
         .then(function clickMetric() {
           return PageObjects.visualBuilder.clickMetric();
         })
+        .then(function addSeries() {
+          return PageObjects.visualBuilder.addSeries();
+        })
+        .then(function addMetric() {
+          return PageObjects.visualBuilder.addMetric();
+        })
+        .then(function selectOverallAverage() {
+          return PageObjects.visualBuilder.selectItem('.vis_editor__series:last-child ' +
```

Comment:
We're really trying to add data-test-subj attributes to elements to avoid cssSelectors like this.  I don't think we have a hard rule to require it, but these are some of the longest selectors I've ever seen.

Reply:
Longest selectors you've ever seen eh? That sound like a challenge!

Reply:
I simplified them a little... but the truth is it needs to "select the aggregation drop down of the last metric of the last aggregation". I'm willing to change it to something simpler if that exists.

Reply:
Input controls has made a bit of progress in this area by creating some useful test functions when dealing with react-select elements.  Check out the *reactSelect* functions starting here:

https://github.com/elastic/kibana/blob/master/test/functional/page_objects/visualize_page.js#L175

Might help simplify the CSS selectors and make it a little easier to read.  Unfortunately, no way we've been able to figure out yet for how to use the data-test-subj field on the options since react-select doesn't seem to offer the ability to pass in custom html attributes for each option.

Reply:
Thx @Stacey-Gammon, I'll refactor to use those.